### ISSUE TITLE
Won't boot with HDMI on newer HD TV sets or Monitors?

### ISSUE THREAD

**Issue by kcraptor82** [petrockblog/RetroPie-Setup#463]
So far I used the latest 2.3 retropie with emulation station and it works great while I have it plugged into my 15" ViewSonic old LCD monitor and a HDMI to DVI cable. Now when I try to plug a regular HDMI cable into the B+ model and try to use it on a newer LED TV or even my Samsung monitor, the raspberry just doesn't show anything past the boot screen. I can see the kernal load, etc, and it trys to load but I can't get into the menu. Strange that it works with a old a$$ LCD screen but not a newer tv?

**Comment by indextwo** [comment]
I'd like to reopen this old thread, because I'm seeing the exact same issue. I've spent _days_ updating my Raspbian from Wheezy to Jessie and getting RetroPie installed and working, the whole time using my RPi with my 24" HD BenQ monitor (going through an HDMI->DVI plug) without issue.

Then, I tried plugging it into my TV, which is a 40" LG, but EmulationStation refuses to get beyond the loading screen. The only difference in the setup is that I moved my RPie from upstairs to downstairs; otherwise, I'm using the same HDMI cable & the same power cable. 

Note that I can boot to Raspbian command line with no problem. This isn't an issue with the RPi's HDMI output, as that works just fine - it's that ES clearly doesn't like my TV. Are there any display-specific configs for ES that need to be changed?

Also note: I did what was mentioned in the StackExchange link above: I added:

    hdmi_force_hoptplug=1
    hdmi_drive=2

Initially I thought this worked - my display output totally change - but I think what _actually_ happened is that, because I jammed my HDMI cable into my TV _after_ restarting my RPi, I seem to have 'fooled' it into working (the display resolution was crazy-low). Howeve,r when I rebooted again, the resolution was back to normal, and emulationstation totally froze after starting.
